samples/resources/README.md

```diff
@@ -0,0 +1,87 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+description: "These samples will show you how to manage reosurce in resource sdk."
+urlFragment: azure-samples-python-management
+---
+
+# Python Management SDK Samples
+
+These samples will show you how to manage reosurces in resource sdk.
+
+## Features
+
+This project framework provides examples for the following services:
+
+### Resource
+* Using the **Management Resource SDK** [azure-mgmt-resource](https://pypi.org/project/azure-mgmt-resource/) for the [Azure Resource Management API](https://docs.microsoft.com/en-us/rest/api/resources/)
```

Comment:
Replace "Management Resource SDK" with:

Azure SDK for Python - Resource Management Library [azure-mgmt-resource](put pypi link)